opencensus/trace/execution_context.py

```diff
@@ -17,8 +17,14 @@
 
 _attrs_slot = RuntimeContext.register_slot('attrs', lambda: {})
 _current_span_slot = RuntimeContext.register_slot('current_span', None)
+_exporter_slot = RuntimeContext.register_slot('is_exporter_thread', False)
 _tracer_slot = RuntimeContext.register_slot('tracer', noop_tracer.NoopTracer())
 
+def is_exporter_thread():
```

Comment:
Curious, why do we need this instead of using `RuntimeContext.is_exporter_thread` directly?

Reply:
1. There should only be a single way to read/modify the same piece of data. It would be quite awkward to read using `RuntimeContext` and write using `execution_context`

2. Consistency with the other `execution_context` slots.

Reply:
> There should only be a single way to read/modify the same piece of data. It would be quite awkward to read using RuntimeContext and write using execution_context

This single way would be `RuntimeContext.is_exporter_thread`.

> Consistency with the other execution_context slots.

We're in the direction to remove execution_context.

Reply:
It would be strange to have execution_context imported but not used.

Reply:
I see. Okay for this PR, we should clean this up once we say goodbye to execution_context.

Reply:
Please clean up https://github.com/census-instrumentation/opencensus-python/blob/1f92449067e2799b299eaa4015ba6eb73795ae34/contrib/opencensus-ext-azure/opencensus/ext/azure/common/transport.py#L45.